velox/functions/sparksql/MapFromArrays.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/expression/VectorFunction.h"
+#include "velox/functions/lib/Map.h"
+
+// Returns a map created using the given key/value arrays.
+// See documentation at https://spark.apache.org/docs/latest/api/sql/#map_from_arrays
+//
+// Example:
+// Select map_from_arrays(array(1,2,3), array('a','b','c'));
+//
+// Result:
+// {1:"a",2:"b",3:"c"}
+
+namespace facebook::velox::functions {
+namespace {
+std::vector<std::shared_ptr<exec::FunctionSignature>> mapFromArraysSignatures() {
+    // array(K), array(V) -> map(K,V)
+    return {exec::FunctionSignatureBuilder()
+            .typeVariable("K")
+            .typeVariable("V")
+            .returnType("map(K,V)")
+            .argumentType("array(K)")
+            .argumentType("array(V)")
+            .build()};
```

Comment:
The signature looks the same with 'MapFunction::signatures()' and shall we reuse that one?